docs/advanced/security-auditing.md

```diff
@@ -268,3 +258,87 @@ services:
         - clutch.service.audit.sink.slack
   // highlight-end
 ```
+
+### Sinks
+
+Sinks asynchronously propagate events to other systems after they are persisted to Clutch's database.
+
+Clutch ships with a logging sink as a scaffold for your own, as well as a sink for Slack.
+
+Adding and customizing audit sinks lets you save or process infrastructure events however appropriate for your needs.
+
+#### Slack Sink
+By default, the Slack sink creates a formatted Slack message using a subset of information saved in an audit event. The default Slack message provides a summary, answering questions such as what operation was performed, who performed the operation, and what resources were operated on.
+
+<img style={ {border: "1px solid black"} } alt="slack audits" src="https://user-images.githubusercontent.com/39421794/112361590-f1431880-8ca9-11eb-9d65-a4a843b6b08c.gif" />
+
+The Slack sink requires your [Slack app’s](https://api.slack.com/start) bot token and the channel to post the messages. You can optionally provide filter rules to control what kinds of Slack audits are sent to the channel.
+
+Example Config:
+```yaml title="backend/clutch-config.yaml"
+...
+services:
+  ...
+  - name: clutch.service.db.postgres
+  ...
+  // highlight-start
+  - name: clutch.service.auditsink.slack
+    typed_config:
+      "@type": types.google.com/clutch.config.service.auditsink.slack.v1.SlackConfig
+      token: <token>
+      channel: <channel>
+      filter:
+        rules:
+          - field: SERVICE
+            text: clutch.k8s.v1.K8sAPI
+  // highlight-end
+  - name: clutch.service.audit
+    typed_config:
+      "@type": types.google.com/clutch.config.service.audit.v1.Config
+      db_provider: clutch.service.db.postgres
+      // highlight-start
+      sinks:
+        - clutch.service.audit.sink.slack
+     // highlight-end
+```
+
+**Custom Slack Messages**
+
+A custom Slack message can be created for a given `/service/method` using the available metadata (the API request and response body) in an audit event. The custom message will then be appended to the default Slack message for a richer Slack audit.
+
+<img style={ {border: "1px solid black"} } alt="slack audits" src="https://user-images.githubusercontent.com/39421794/112363388-d2458600-8cab-11eb-80d6-1324c5e5b64a.png" />
+
+**Creating a Custom Slack Messages**
+
+The feature is powered by the Golang [Template](https://golang.org/pkg/text/template/) package. In the clutch-config, you can provide a template with the field names from the API request and/or response, which will be replaced with the field values at parse time. The template can also include Slack [`mrkdwn`](https://api.slack.com/reference/surfaces/formatting#basics) to add useful visual highlights to the custom message.
+
+Creating the template:
```

Comment:
didn't mention that we also have custom helpers to help format the template since we just have `slackList` atm, thought I can mention it and add that we plan to add more template helpers?